library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/TransactionPayloadFragment.kt

```diff
@@ -136,6 +122,28 @@ internal class TransactionPayloadFragment :
         super.onCreateOptionsMenu(menu, inflater)
     }
 
+    private fun shouldShowSaveIcon(transaction: HttpTransaction?) = when {
+        // SAF is not available on pre-Kit Kat so let's hide the icon.
+        (Build.VERSION.SDK_INT < Build.VERSION_CODES.KITKAT) -> false
+        (type == TYPE_REQUEST) -> {
+            (0L == (transaction?.requestContentLength ?: 0L))
```

Comment:
Sorry, but I suggested a mistake to you. It should be:
```suggestion
            (0L != (transaction?.requestContentLength ?: 0L))
```